tests/php/test-class-amp-comments-sanitizer.php

```diff
@@ -33,138 +34,233 @@ class Test_AMP_Comments_Sanitizer extends TestCase {
 	public function setUp() {
 		parent::setUp();
 		$GLOBALS['post'] = self::factory()->post->create_and_get();
-		$this->dom       = new Document();
+
+		$GLOBALS['wp_scripts'] = null;
+	}
+
+	public function tearDown() {
+		parent::tearDown();
+
+		$GLOBALS['wp_scripts'] = null;
```

Comment:
Generally, it's always better to reset the changes made before calling the parent `tearDown()` so as to not inadvertently affect the test environment (this becomes more apparent when we switch to the snake case `set_up*()` and `tear_down*()` variant due to the changes in the [Core test suite](https://make.wordpress.org/core/2021/09/27/changes-to-the-wordpress-core-php-test-suite/)).

```suggestion
		$GLOBALS['wp_scripts'] = null;

		parent::tearDown();
```